indexes/subset-offset-and-size.go

```diff
@@ -0,0 +1,21 @@
+package indexes
+
+import "errors"
+
+type SubsetOffsetAndSize struct {
+	Subset uint64 // uint24, 3 bytes, max 16.7 MB (megabytes)
```

Comment:
could be even smaller? uint16 ? but might not be worth it and better to have an extra byte

Reply:
Yeah I don't have a strong opinion here. @gagliardetto ?

Reply:
yeah uint16 should be eough